Christopher-christpheranderson/homework-1/fizzbuzz/js/main.js

```diff
@@ -0,0 +1,12 @@
+for (i = 1; i < 100; i++){
+	if(i % 3){ 
+		console.log("fizz")
+	}
+	/*else if{ 
+	console.log([i])*/
+	else if(i % 5){
+		console.log("buzz")
+	}	else if( i % 5 && i % 3){
```

Comment:
If you run this code in the browser, you should notice an issue with the output -- you'll never see 'fizzbuzz' logged! Think about the structure here and where the final statement would need to be evaluated in order for it to be part of the output! (This is the biggest challenge of this exercise!)